.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
Yes it was failing with error with Java incompatibility 52. I tested the build with Travis on my fork and cured it before sending PR. Should I point log here?